README.md

```diff
@@ -16,17 +16,23 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
-```
+```the purpose of a clearfix in CSS is to fix the floating problem. It clear it's child element so no need to add additional markup. ``
 
+img {
+	float: right;
+}
+
+div {
+	clear: both;
+}
 ### Question #2
 
 What does the following selector do?  `ul.dropdown > li`?
 
 Select 1:
 ```
-[] Selects all li's which are directly inside a ul of class dropdown (children)
-[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[X] Selects all li's which are directly inside a ul of class dropdown (children)
+[X] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
 [] Selects all ul's of class dropdown, as well as the children elements that are li's
 [] Selects all ul's of class dropdown, only if their children are exclusively li's
 ```
```

Comment:
http://code.tutsplus.com/tutorials/the-30-css-selectors-you-must-memorize--net-16048